tests/product/test_connectors.py

```diff
@@ -61,6 +61,96 @@ def test_basic_connector_add_remove(self):
             self.assert_has_default_connector(host)
         self._assert_connectors_loaded([['system'], ['tpch']])
 
+    def test_basic_connector_add_remove_coord_using_dash_h(self):
+        self.setup_cluster(NoHadoopBareImageProvider(),
+                           self.STANDALONE_PRESTO_CLUSTER)
+        self.run_prestoadmin('server start')
+        for host in self.cluster.all_hosts():
+            self.assert_has_default_connector(host)
+
+        self._assert_connectors_loaded([['system'], ['tpch']])
+
+        self.run_prestoadmin('connector remove tpch -H %(master)s')
+        self.run_prestoadmin('server restart')
+        self.assert_path_removed(self.cluster.master,
```

Comment:
I see what you mean about -H not making sense.  It's really weird behavior to delete the presto-admin connector config file when you are only removing the connector on one node.  But I still stand by what I said before.